homeassistant/components/luftdaten/__init__.py

```diff
@@ -40,6 +40,10 @@
 SENSOR_PRESSURE_AT_SEALEVEL = "pressure_at_sealevel"
 SENSOR_TEMPERATURE = "temperature"
 
+SENSOR_LAEQ = "noise_LAeq"
+SENSOR_LA_MIN = "noise_LA_min"
```

Comment:
```suggestion
SENSOR_LA_MIN = "noise_la_min"
```

Reply:
"value_type" is noise_LAeq, noise_LA_min and noise_LA_max as defined by luftdaten.info api. Not sure if noise_la_eqeq, noise_la_min and noise_LA_max will work.